climada/entity/exposures/litpop/litpop.py

```diff
@@ -508,11 +515,11 @@ def set_custom_shape(self, shape, total_value, res_arcsec=30, exponents=(1,1),
             self.meta = {'crs': self.crs}
 
     @staticmethod
-    def _set_one_country(country, res_arcsec=30, exponents=(1,1), fin_mode=None,
+    def _from_country(country, res_arcsec=30, exponents=(1,1), fin_mode=None,
                          total_value=None, reference_year=DEF_REF_YEAR,
                          gpw_version=GPW_VERSION, data_dir=SYSTEM_DIR):
         """init LitPop exposure object for one single country
-        See docstring of set_countries() for detailled description of parameters.
+        See docstring of from_countries() for detailled description of parameters.
```

Comment:
Replace all occurrences of the deprecated `set_`-method, not only in the code, but also in the docstrings and comments.